resources/ru/cemu.po

```diff
@@ -152,921 +1146,1090 @@ msgstr ""
 "Продолжайте действовать только в том случае, если вы готовы рискнуть "
 "потерять онлайн-доступ к Вашим Wii U и / или NNID."
 
-#: GeneralSettings2.cpp:270
+#: GeneralSettings2.cpp
 msgid "Account settings"
 msgstr "Настройки аккаунта"
 
-#: GeneralSettings2.cpp:277
+#: GeneralSettings2.cpp
 msgid "Active account"
 msgstr "Активный аккаунт"
 
-#: GeneralSettings2.cpp:289
+#: GeneralSettings2.cpp
+msgid "Create"
+msgstr "Создать"
+
+#: GeneralSettings2.cpp InputSettings.cpp
+msgid "Delete"
+msgstr "Удалить"
+
+#: GeneralSettings2.cpp
+msgid "Enable online mode"
+msgstr "Включить онлайн-режим"
+
+#: GeneralSettings2.cpp
 msgid "Account information"
 msgstr "Информация об аккаунте"
 
-#: GeneralSettings2.cpp:297
-msgid "Id"
-msgstr "Id"
+#: GeneralSettings2.cpp
+msgid "The persistent id is the internal folder name used for your saves"
+msgstr ""
+"Постоянный идентификатор - это имя внутренней папки, используемой для ваших "
```

Comment:
Below you kept "persistent id" as it is - which is fine, just decide which translation to use.